core/src/main/scala/com/codacy/analysis/core/upload/ResultsUploader.scala

```diff
@@ -52,8 +52,19 @@ class ResultsUploader private (commitUuid: String, codacyClient: CodacyClient, b
 
   def sendResults(toolResults: Seq[ResultsUploader.ToolResults],
                   metricsResults: Seq[MetricsResult]): Future[Either[String, Unit]] = {
```

Comment:
Why is MetricsResult in a different scope @xplosunn

Reply:
Because it's directly what we send to the API. It's in the same scope as those. `ResultsUploader.ToolResults` is an intermediate representation, which is why it lies here.